packages/pricing/src/repositories/price-rule.ts

```diff
@@ -76,20 +76,23 @@ export class PriceRuleRepository extends DALUtils.MikroOrmBaseRepository {
     const manager: SqlEntityManager =
       this.getActiveManager<SqlEntityManager>(context)
 
-    const toCreate = await Promise.all(data.map(async (ruleData) => {
-      const ruleDataClone = { ...ruleData } as CreatePriceRuleDTO & {
-        rule_type: string
-        price_set: string
-        price_set_money_amount: string
-      }
-      ruleDataClone.rule_type = ruleData.rule_type_id
+    const toCreate = await Promise.all(
```

Comment:
todo: there no async action in there, should we refactor to a simple `data.map(() => {})`